### ISSUE TITLE
Hide & Show Controller

### ISSUE THREAD

**Issue by esafirm** [bluelinelabs/Conductor#191]
Hi i want to switch between `Controller` just like hide & show in one `Router` without losing my reference to the views.
Is it possible? 

Thanks.

**Comment by EricKuck** [comment]
You can retain the view of the initial controller by setting its retainViewMode to `RETAIN_DETACH`. The controller that's getting pushed, then popped on top of that view is actually getting completely destroyed though, so there's no way to save that view.

**Comment by esafirm** [comment]
I see. The problem is that every router have their own stack.

My current implementation is to have multiple `Router` and handle the `hide & show` with View Animation and Visibility
It's going nice, just wondering that i can be handled by Conductor 😄 

Thanks for your time. You can close this anytime.

**Comment by EricKuck** [comment]
Yeah, that's about the best way to do it for now. Having two views that are equal in the hierarchy added to the same router would be nice for certain cases, but I don't believe the added complexity is worth it right now. Just so we're clear, you're basically looking for the Conductor version of ViewFlipper, right?

**Comment by esafirm** [comment]
Yep, `ViewFlipper` with click trigger to be precise.

**Comment by stephanschuster** [comment]
Maybe this helps:

https://github.com/bluelinelabs/Conductor/issues/260#issuecomment-295856755
https://github.com/bluelinelabs/Conductor/issues/156
